src/new-components/hidden-visually/hidden-visually.module.css

```diff
@@ -0,0 +1,6 @@
+.hiddenVisually {
+    width: 1px;
+    height: 1px;
+    clip: rect(1px, 1px, 1px, 1px);
+    white-space: nowrap;
```

Comment:
It's kind of amazing that browsers still don't allow us to do this in a more classy fashion. 😅
Made me go read a bit about `visibility:collapse` which is still not a thing.